source/unit.js

```diff
@@ -559,7 +564,7 @@ export class Unit {
    * @param amt the quantity of this unit to be converted
    * @param massUnit the target/to unit for which the converted # is wanted
    * @param molecularWeight the molecular weight of the substance for which the
-   *  conversion is being made
+   * conversion is being made
```

Comment:
We usually indent wrapped @param lines by an extra space so the '@' sticks out and it is easier to see the @params.